src/common/components/Toggle/Toggle.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+
+import Toggle, { TToggleProps } from './Toggle'
```

Comment:
@sanjanadhikari1 it's good to import from index.ts instead of Toggle.tsx

Reply:
yes correct, will fix it asap. thanks for commenting